mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/GroupRegistration.java

```diff
@@ -143,7 +142,7 @@ private Disposable consumeWorkQueue() {
 
         return delayGenerator.delayIfHaveTo(currentRetryCount)
             .publishOn(Schedulers.elastic())
```

Comment:
Needed?

This could be redundant with the ReactiveMailboxListenerWrapper. Could it cause some slowness?